app/routes/participate.js

```diff
@@ -8,6 +8,14 @@ export default Ember.Route.extend(WarnOnExitRouteMixin, {
   store: Ember.inject.service(),
   currentUser: Ember.inject.service(),
 
+  _getCondition(profileId) {
+    var participantId = profileId.split('.')[1];
+    var id = participantId.split('_')[1];
+    if (id % 2 === 0) {
+      return '7pm';
+    }
+    return '10am';
```

Comment:
Yup, I did that here: https://github.com/CenterForOpenScience/exp-addons/pull/117/files#diff-7ef242c3954ea3f85b53c4cabf77d4b1R32 

And in this case I checked the condition to show the correct question: https://github.com/CenterForOpenScience/exp-addons/pull/117/files#diff-9150b292474824ef67f55d3899fd974aR556